engine/link/link.asm

```diff
@@ -772,7 +772,7 @@ Link_PrepPartyData_Gen2:
 ; Fill 5 bytes at wc8f4 with $20
 	ld de, wc8f4
 	ld a, $20
-	call Function285d3
+	call Link_Fill5Bytes
 
 ; Copy all the mail messages to wc9f9
```

Comment:
This is a leftover from Crystal; in Gold, these addresses are $100 lower. This should be `wc8f9`. (Likewise with `wc9bf` instead of `wcabf` later below.)

Reply:
The addresses match:
```
00:c8f4 wc8f4
00:c8f9 wc8f9
...
00:c9bf wc9bf
```

Reply:
They do; note that I'm referring to the address in the comment. This block of code starts writing right after the copy above, which writes 5 bytes to $c8f4 and leaves an updated pointer in `de`; naturally, the next write will start at $c8f9.